raster.py

```diff
@@ -0,0 +1,337 @@
+###############################################################################
+# MIT License
+# 
+# Copyright (c) 2023 The Nature Conservancy - Brazil
+# 
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# 
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+###############################################################################
+from dataclasses import dataclass
+from osgeo import gdal, gdal_array
+import numpy as np
+from typing import List
+import matplotlib.pyplot as plt
+import math
+import glob
+import os
+import matplotlib.animation as animation
+
+GDRIVE_BASE = ""
+RASTER_BASE = ""
+MODEL_BASE = ""
+SAMPLE_DATA_BASE = ""
+ANIMATIONS_BASE = ""
+TEST_DATA_BASE = ""
+MOUNTED = False
```

Comment:
These globals are used within raster and should be parameters. Do you mind if I follow up with that in a separate CL to avoid too many changes in a single PR?

Reply:
Can you clarify how this would work currently? Are values injected somehow?

Reply:
I've initialized these to reasonable defaults which most of us use until I parameterize them.